oneview/data_source_task.go

```diff
@@ -226,6 +226,10 @@ func dataSourceTaskRead(d *schema.ResourceData, meta interface{}) error {
 		taskList, err := config.ovClient.GetTasks(filter, "", "", "", "", "")
 		if err != nil {
 			d.SetId("")
+			return err
+		} else if taskList.URI.IsNil() {
+			d.SetId("")
+			return nil
 		}
```

Comment:
This  need to be corrected. 

The URI to be checked is 

if err != nil || task.URI.IsNil() {
		d.SetId("")
		return nil
	}

Reply:
done